aiven/client/cli.py

```diff
@@ -1430,6 +1433,7 @@ def service__user_password_reset(self):
     @arg("--redis-acl-keys", help="ACL rules for keys")
     @arg("--redis-acl-commands", help="ACL rules for commands")
     @arg("--redis-acl-categories", help="ACL rules for command categories")
+    @arg("--redis-acl-channels", help="ACL rules for channels (Redis only)")
```

Comment:
Here we have `(Redis only)` in the end, but not arg's from above, and I wonder whether we should make it consistent with the others (adding this to every Redis related argument)?

Reply:
I made it consistent with other params (typo due to copy/paste from another place). Command itself is marked as Redis-only help="Set Redis service user access control").